modules/core/src/test/java/org/apache/ignite/internal/testframework/IgniteTestUtils.java

```diff
@@ -164,6 +164,36 @@ public static Object getFieldValue(Object target, Class<?> declaredClass, String
         }
     }
 
+    /**
+     * Returns field value.
+     *
+     * @param obj target object from which to get field value ({@code null} for static methods)
+     * @param fieldName name of the field
+     * @return field value
+     */
+    public static Object getFieldValue(Object obj, String fieldName) {
+        Class<?> cls = obj.getClass();
+        Exception ex = null;
+
+        while (cls != Object.class) {
+            try {
+                return getFieldValue(obj, obj.getClass(), fieldName);
+            } catch (Exception ex0) {
+                cls = cls.getSuperclass();
```

Comment:
'cls' is not used.
Do you actually mean this?
```suggestion
                return getFieldValue(obj, cls, fieldName);
            } catch (Exception ex0) {
                cls = cls.getSuperclass();
```

Reply:
Fixed. Move test utils method from 2.x